### DISCUSSION TITLE
Drag and Drop delay and/or drag to offset

### DISCUSSION THREAD

**Discussion by alanreay** [codeceptjs/CodeceptJS#3111] [Q&A]
Been having a lot of issues with Drag and Drop.  WebDriver just didn't work at all (would drag to the cursor location every time, not the element location), so switched to Playwright.  This works better, but still fails frequently.  Some of the issues appear to be that it doesn't hold the item over the drop zone long enough.  

Does anyone know of a way to delay the drag and drop over the drop element?

Another needed item would be to offset the drop location.  If we could specify the drop location (default, center, top left, etc) or give an offset in pixels, this would also help a great deal.

**Comment by kobenguyent** [comment]
https://codecept.io/helpers/Playwright/#draganddrop now provides more options.
